src/desktop/apps/unsupported_browser/template.jade

```diff
@@ -1,28 +1,29 @@
-    link( type='text/css', rel='stylesheet', href=asset('/assets/unsupported_browser.css') )
+    link( type='text/css' rel='stylesheet', href=asset('/assets/main_layout.css') )
+    link( type='text/css' rel='stylesheet', href=asset('/assets/unsupported_browser.css') )
   body.body-header-fixed.minimal-header
-    #main-layout-container
-      #unsupported-browser.main-layout-container
-        img.unsupported-browser-artsy-logo( src="images/full_logo.png" )
-        .unsupported-browser-message
-          p #{sd.BROWSER.family} #{sd.BROWSER.major} is not supported.
-          p Please update your browser.
-        .unsupported-browser-recommendations
-          a.unsupported-browser-update-first.avant-garde-button-black.settings-submit( href="https://www.google.com/intl/en/chrome/browser/" )
-            | Chrome
-          a.avant-garde-button-black.settings-submit( href="http://www.apple.com/safari" )
-            | Safari
-          a.unsupported-browser-update-first.avant-garde-button-black.settings-submit( href="https://www.mozilla.org/en-US/firefox/new/?utm_source=artsy-net&utm_medium=referral")
-            | FireFox
-          a.avant-garde-button-black.settings-submit( href="https://www.microsoft.com/en-us/windows/microsoft-edge" )
-            | Microsoft Edge
+    main#main
+      #main-layout-container
+        #unsupported-browser.main-layout-container
+          img.unsupported-browser-artsy-logo( src="images/full_logo.png" )
+          .unsupported-browser-message
+            p #{sd.BROWSER.family} #{sd.BROWSER.major} is not supported.
+            p Please update your browser.
+          .unsupported-browser-recommendations
+            a.unsupported-browser-update-first.avant-garde-button-black.settings-submit( href="https://www.google.com/intl/en/chrome/browser/" )
+              | Chrome
+            a.avant-garde-button-black.settings-submit( href="http://www.apple.com/safari" )
+              | Safari
+            a.unsupported-browser-update-first.avant-garde-button-black.settings-submit( href="https://www.mozilla.org/en-US/firefox/new/?utm_source=artsy-net&utm_medium=referral")
+              | FireFox
+            a.avant-garde-button-black.settings-submit( href="https://www.microsoft.com/en-us/windows/microsoft-edge" )
+              | Microsoft Edge
 
-        br
-        br
+          br
+          br
 
-        form.unsupported-browser-continue( action='', method='post' )
-          input( type="hidden", name="redirect-to", value=sd.UNSUPPORTED_BROWSER_REDIRECT || '/' )
-          button.avant-garde-button-black.unsupported-browser-submit.is-block Continue with #{sd.BROWSER.family} #{sd.BROWSER.major}
+          form.unsupported-browser-continue( action='' method='post' )
+            input( type="hidden" name="redirect-to", value=sd.UNSUPPORTED_BROWSER_REDIRECT || '/' )
+            button.avant-garde-button-black.unsupported-browser-submit.is-block Continue with #{sd.BROWSER.family} #{sd.BROWSER.major}
```

Comment:
The only thing I did in these jade templates was wrap the top level in `main#main`. We actually can't use the `main-layout-container` id as a fragment here because it's got really wild styling applied to it depending on the route, so using it in the new app shell creates big margins where we don't want them. There's some prettier diffing also, surprisingly.